src/engine/strat_engine/backstore/util.rs

```diff
@@ -78,6 +80,7 @@ pub fn get_stratis_block_devices() -> StratisResult<Vec<PathBuf>> {
 
     let devices: Vec<PathBuf> = enumerator
         .scan_devices()?
+        .filter(|dev| dev.is_initialized())
```

Comment:
The comment and TODO below are invalidated by this line, because now a third explanation is that stratisd is starting up so early that the devices are not yet initialized. So, it should be fixed up, and I think that the TODO might as well be removed, since under these circumstances it is probably the case that the block of code that it annotates can never be removed.

Reply:
I agree the comment could use an update, but I don't think we want to keep the code it annotates.  For example it's entirely possible that a big configuration at boot has **many** block devices.  If we walk the udev db before any are initialized and thus find none, we then would fall into this annotated code which would falsely think that everything is empty, thus causing block reads for every device looking for Stratis signatures.  IMHO it's OK that at the point in time that we scan things are empty as we will get them when events are generated.  I'm leaning towards removing this code and removing whatever tests that fail on older distribution that run on Travis CI.